how_to_solve_a_problem.md

```diff
@@ -0,0 +1,90 @@
+ # How to Solve a Problem
+
+Mastering problem solving will get you far. There are some techniques to do it, and it's important to be aware of the fact your day is mostly composed of problems that need to be solved.
+
+A dev who encounters a nasty bug; An account manager working on a price estimation; A sales person trying to reach more leads. These are all big problems, which probably have solutions.
+
+As we will refer to "problem" as "bug" this chapter might seem as relevant only for developer. However the methods and mind set applies to all the roles in the company.
+
+## Finding the cause
+
+Understanding the cause of the bug is the first step and the most important one. Only in very rare cases where a serious bug happens in production and we want to first patch the bug with some quick fix, we will start by working around the problem. But we'll always circle back at finding the cause.
+
+It's better to reflect on the term "the cause". Are you sure the cause you are seeing is the "root cause"?
+Systems are always convoluted, and miss-identification of the real cause, will result with the bug simply appearing in different places.
+
+Assumptions are good but need to be taken with a pinch of salt. You shouldn't dive in too deeply just based on your assumption without verifying every step. For example, don't waste time on debugging line by line a certain module before you have asserted that it's indeed the one to blame.
+
+The first step for diving into a bug is not using a scalpel, but rather a D9. This means that the first steps don't require slow and tedious examniation of lots of code, but rather big and radical changes. Examples:
+
+| Problem | Solution |
+| -- | -- |
```

Comment:
For me it looks ok:

![how_to_solve_a_problem_md__12-solve-problems__-_gitbook_editor](https://cloud.githubusercontent.com/assets/125707/16194294/866717ee-36fc-11e6-8af6-dadfabf42212.jpg)